tests/js/spec/views/settings/organizationDeveloperSettings/index.spec.jsx

```diff
@@ -20,6 +20,7 @@ describe('Organization Developer Settings', function() {
   const routerContext = TestStubs.routerContext();
 
   const publishButtonSelector = 'StyledButton[icon="icon-upgrade"]';
+  const deleteButtonSelector = 'StyledButton[icon="icon-trash"]';
```

Comment:
Can we use `aria-label` selector to select the button? Doing this in tests (especially for interactive elements like a button) can help train us to build more accessible apps.